apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -246,7 +246,7 @@ export const EventSetupTab = (
                 </Trans>
               </div>
             )}
-            {validLocations.length > 0 && validLocations.length !== locationOptions.length && (
+            {validLocations.length > 0 && (
```

Comment:
This check was actually there to stop showing 'Add a Location' CTA when all possible locations are added. Something has broken it. But for now it is acceptable.